components/backup/tests/integrations/test_backup.rs

```diff
@@ -621,15 +621,18 @@ fn test_backup_raw_meta() {
     let mut checksum = 0;
     let mut total_kvs = 0;
     let mut total_bytes = 0;
+    let mut total_size = 0;
     for f in files {
         checksum ^= f.get_crc64xor();
         total_kvs += f.get_total_kvs();
         total_bytes += f.get_total_bytes();
+        total_size += f.get_size();
     }
     assert_eq!(total_kvs, key_count + 1);
     assert_eq!(total_kvs, admin_total_kvs);
     assert_eq!(total_bytes, admin_total_bytes);
     assert_eq!(checksum, admin_checksum);
+    assert!(total_size > 0);
```

Comment:
I think we should use an exact size for this test.

Reply:
```suggestion
    assert_eq!(total_size, 1611); // update this number if needed
```

@zhangjinpeng1987 the exact size relies on the SST encoding which can be considered a magic number. Is this change acceptable?

Reply:
For the specified version of RocksDB, it should be constant.